vignettes/adrs.Rmd

```diff
@@ -788,6 +794,30 @@ dataset_vignette(
 )
 ```
 
+## Derive `AVAL` for New Parameters {#aval}
+
+For cases where `AVALC` has been derived for new parameters above as `"Y"` or `"N"`,
+we need to set `AVAL` to numeric versions such as `1`/`0`.
+
+```{r eval=TRUE}
+adrs <- adrs %>%
+  mutate(
+    AVAL = case_when(
+      AVALC == "Y" ~ 1,
+      AVALC == "N" ~ 0,
+      TRUE ~ AVAL
+    )
+  )
+```
+
+```{r, eval=TRUE, echo=FALSE}
+dataset_vignette(
+  adrs,
+  display_vars = vars(USUBJID, PARAMCD, AVALC, AVAL),
+  filter = USUBJID == "01-701-1015" & AVALC %in% c("Y", "N")
```

Comment:
Should we display all parameters to demonstrate that `AVAL` for non-binary parameter is not changed?

Reply:
no i think it's ok and users would assume that i'm sure.